src/iso19111/coordinateoperation.cpp

```diff
@@ -106,6 +106,12 @@ constexpr double UTM_SOUTH_FALSE_NORTHING = 10000000.0;
 static const std::string INVERSE_OF = "Inverse of ";
 static const char *NULL_GEOCENTRIC_TRANSLATION = "Null geocentric translation";
 static const char *NULL_GEOGRAPHIC_OFFSET = "Null geographic offset";
+static const char *APPROXIMATE_TRANSFORMATION_PREFIX =
+    " (approximate transformation";
+static const char *APPROXIMATE_TRANSFORMATION = " (approximate transformation)";
```

Comment:
just to be sure, this "approximate transformation" is the same as the "rough transformation"? If so, then I guess make all of other instances of "rough" to "approximate".